lib/omnisharp-atom/linter.ts

```diff
@@ -84,6 +84,13 @@ class LinterCSharp extends Linter.Linter {
             });
         }
     }
+
+    private getFrameworks(error: OmniSharp.Models.DiagnosticLocation): string {
+        var frameworks = _.map(error.Projects, (project: string) => {
+            return project.split('+')[1];
+        }).filter((fw: string) => fw.length > 0);
+        return frameworks.join(',');
+    }
```

Comment:
Want to add this method to the prototype of OmniSharp.Models.DiagnosticLocation. But as a typescript noob don't know where we can do this. Any suggestions @david-driscoll